adli.py

```
@@ -0,0 +1,28 @@
+
+    args_parser.add_argument(
+        "source",
+        type=str,
+        help="Path to source file"
+    )
+    
+    parsed_args = args_parser.parse_args(argv[1:])
+    source = parsed_args.source
+
+    try:
+        open(source)
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)